### PR TITLE
Feature/improve translations

### PR DESCRIPTION
**Pull request by HLXEasy**
- Updated all translations with their current state from Transifex
- Improved internal handling of supported languages
- Sorted languages selection list within the application
- Added some new languages

### CODE REVIEW

File: lib/supported_locales.dart [hunk 1]
```diff
@@ -1,33 +1,46 @@
 import 'dart:ui';
 
 const supportedLocales = const [
+  // Base language:
```
Comment (azbuky) [inline lib/supported_locales.dart:4]: This is the wrong place to add new languages. Check lib/settings/available_languages.dart
This file will be removed as it is unused.

File: lib/contacts/contact_details.dart [hunk 2]
```diff
@@ -49,9 +49,9 @@ class ContactDetails extends HookConsumerWidget {
         context,
         l10n.removeContact,
         l10n.removeContactConfirmation(contact.name),
-        l10n.yesUppercase,
```
Comment (azbuky) [inline lib/contacts/contact_details.dart:52]: .toUppercase() does not work for all languages. Translations should handle lower case, capital case, and upper case separately. Please revert these changes.

File: lib/supported_locales.dart (hunk 1 above)
Comment (HLXEasy) [reply lib/supported_locales.dart:4]: Thx, also figured it out already.

File: lib/contacts/contact_details.dart (hunk 2 above)
Comment (HLXEasy) [reply lib/contacts/contact_details.dart:52]: That's odd because within the translation there's no context available. But if it works like that, I will revert it of course.

Comment (HLXEasy) [reply lib/contacts/contact_details.dart]: Reverted with [8f347b0](https://github.com/azbuky/kaspium_wallet/pull/76/commits/8f347b03f4a684662bdd5fd831702a7401a7b80b)
, [0a37c7a](https://github.com/azbuky/kaspium_wallet/pull/76/commits/0a37c7a7f427855f88844d46e3d78732524ee955)
 and [13ba46a](https://github.com/azbuky/kaspium_wallet/pull/76/commits/13ba46a13b3a06d744ec93d9fea1287d557dbfbf)

**Comment by HLXEasy** [comment]
After the revert of the upper-/lowercase-change, we should wait until the translators of the already finished translations updated these last four strings. After that it might be a good state to approve. If it's fine for you too, of course...
